docs/UserGuide.md

```diff
@@ -48,13 +48,13 @@ McScheduler is a **one-stop solution for McDonald's Shift Managers to manage shi
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `worker delete WORKER_INDEX`, `WORKER_INDEX` is a parameter for the index of the worker (in the worker list) you want to delete from the application which can be used as `worker delete 4`.
+  e.g. in `worker-delete WORKER_INDEX`, `WORKER_INDEX` is a parameter for the index of the worker (in the worker list) you want to delete from the application which can be used as `worker-delete 4`.
 
 * Items in square brackets are optional.<br>
   e.g `n/NAME [r/ROLE]` can be used as `n/John Doe r/Cashier` or as `n/John Doe`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[r/ROLE-NUMBER_NEEDED]…​` can be used as ` ` (i.e. 0 times), `r/Cashier-3`, `r/Cashier-1 r/Cleaner-2` etc.
+  e.g. `[r/ROLE <space> NUMBER_NEEDED]…​` can be used as ` ` (i.e. 0 times), `r/Cashier 3`, `r/Cashier 1 r/Cleaner 2` etc.
```

Comment:
`<space>` can be removed to be consistent with the rest of the commands

Reply:
Agreed, I have updated User Guide